gtfparse/attribute_parsing.py

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015. Mount Sinai School of Medicine
+# Copyright (c) 2015-2016. Mount Sinai School of Medicine
```

Comment:
Not sure if we need it, but nominally seems to extend the copyright forward to last edited date: http://programmers.stackexchange.com/questions/124687/copyright-statements-in-source-code